articles/password-checker-javascript/index.md

```diff
@@ -0,0 +1,251 @@
+
+We are going to have three levels to denote how secure a password is. They are:
+
+- **Strong**: The password has to meet all the requirements.
+
+    Using the metrics above, we are going to create a strong level which has at least one lowercase letter (`?=.*[a-z]`), one uppercase letter (`?=.*[A-Z]`), one digit (`?=.*[0-9]`), one special character (`?=.*[^A-Za-z0-9]`), and is atleast 8 characters long(`?=.{8,}`).
+
+    ```JavaScript
+    (?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^A-Za-z0-9])(?=.{8,})
+    ```
+
+    ![Strong password](/engineering-education/password-checker-javascript/strong-password.png)
+
+- **Medium**:  If the password is at least 6 characters long and meets all the other requirements, or has no digit but meets the rest of the requirements.
+
+    The code is the same as for the **Strong** level only that **?=.{6,}** shows that we are checking for at least 6 characters. It also has **|** for checking either of the two conditions `(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^A-Za-z0-9])(?=.{6,})` or `(?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9])(?=.{8,})`.
+
+    ```JavaScript
+    ((?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^A-Za-z0-9])(?=.{6,}))|((?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9])(?=.{8,}))
+    ```
+
+    ![Medium password](/engineering-education/password-checker-javascript/medium-password.png)
+
+- **Weak**: If the entered password does not meet the strong or medium-level requirements, then it is deemed weak.
+
+    ![Weak password](/engineering-education/password-checker-javascript/weak-password.png)
+
+#### HTML Code and explanation
+
+```HTML
+<html>
+<head>
+    <!-- Bootstrap 5 link -->
+    <link href= "https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- CSS Styling -->
+    <style>
+        .passwordInput{
+            margin-top: 5%; 
+            text-align :center;
+        }
+
+        .displayBadge{
+            margin-top: 5%; 
+            display: none; 
+            text-align :center;
+        }
+    </style>
+</head>
+<body>
+    <div class="d-flex text-danger justify-content-center">
+        <h4 style="margin-top: 10%; text-align :center">
+            Password Strength Checker
+        </h4>
+        <br>
+        <input type="password" placeholder="Type your password" id="PassEntry" class="form-control passwordInput">
+        <br>
+        <span id="StrengthDisp" class="badge displayBadge">Weak</span>
+        <br>
+    </div>     
+</body>
+</html>
+```
+
+- We'll use Bootstrap 5 for styling. We'll use a CDN link for the bootstrap file and link it to the HTML file using the `<link>` tag. 
+
+- We'll write our custom styles using the `<style>` tag. We have two classes and their respective stylings.
```

Comment:
Hi @Agusioma 
Minor suggestion. Add info about which elements you are writing the custom styles for.